com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -328,6 +331,12 @@ public float GetCumulativeReward()
             return m_CumulativeReward;
         }
 
+        void UpdateRewardStats()
+        {
+            var gaugeName = $"{m_PolicyFactory.behaviorName}.CumulativeReward";
```

Comment:
Could do this inline in NotifyAgentDone, but I figured I'd keep it closer to the reward logic.